Makefile

```diff
@@ -76,3 +76,9 @@ deploy: manifests ## Deploy controller to the K8s cluster specified in ~/.kube/c
 
 undeploy: ## Undeploy controller from the K8s cluster specified in ~/.kube/config.
 	$(KUSTOMIZE) build config/default | $(KO) delete -f -
+
+##@ Publish
+
+publish:
+	echo "${AUTH_TOKEN}" | $(KO) login ghcr.io --username ${REPOSITORY_OWNER} --password-stdin
+	$(KO) publish -B --tags ${RELEASE_TAG} ./
```

Comment:
`${RELEASE_TAG}` should have a default specified by the Makefile.

A common pattern is to define the variable with a default. Something like:

```
VERSION ?= "dev"
```

Reply:
Done. 240399d51dca528f3f5341dc5e629473ed34415d